muchlisinadi/DSC_UI_GAN
# Bugfix Notebook Week 1

Bugfix
- Di cell terakhir, ada variable `bs`. Switch ke `batch_size` dengan asumsi typo

Modifs (based on [eriklindernoren/Pytorch-GAN implementations](https://github.com/eriklindernoren/PyTorch-GAN/blob/master/implementations/gan/gan.py))
- unused `test_dataset`, removed
- using zero means and unit variance (see `transform`)
- switch sample data di awal dengan data dari `train_dataset`
- switch `train_dataset.train_data.size` => `train_dataset.data.size` (dari warning)
- removed argument `x` di function `G` (G dapat data dari generate sendiri)
